deployment/roles/oracle/defaults/main.yml

```diff
@@ -3,5 +3,4 @@ bridge_path: "/home/{{ compose_service_user }}/bridge"
 ALLOW_HTTP: no
 QUEUE_URL: amqp://rabbit
 REDIS_URL: redis://redis
-syslog_server_port: udp://127.0.0.1:514
```

Comment:
I have discovered that it was already in https://github.com/poanetwork/deployment-bridge/blob/master/bridge-nodejs/roles/logging/defaults/main.yml. Could you confirm that was incorrect too?

Reply:
That is my understanding. @ArseniiPetrovich @phahulin What do you think?

Reply:
`udp://127.0.0.1:514` is syslog service on localhost.
I deployed a test node without `syslog_server_port` and it logs to local syslog anyway, so this change doesn't seem to break anything. But I'd like to ask @ArseniiPetrovich for his opinion too.